### DISCUSSION TITLE
Unknown protocol "rtmp" in URI "rtmp://mysite.com/live/test"!

### DISCUSSION THREAD

**Discussion by ngivoni** [savonet/liquidsoap#2504] [User Support]
I am attempting to create a playlist with the an RTMP input and have it output as an RTMP (pass through RTMP):

Playlist:
rtmp://mysite.com/live/test

LiquidSoap script:
source = playlist("playlist.pls")
enc = %ffmpeg(
  format="flv",
  %audio.copy,
  %video.copy)

url = "rtmp://127.0.0.1:1935/live/test"
output.url(url=url, enc, source,fallible=true)

When I do so I'm receiving the following error: 
Unknown protocol "rtmp" in URI "rtmp://mysite.com/live/test"!

Is LiquidSoap missing some library?

**Comment by toots** [comment]
`playlist` handles request-based sources and operates on downloaded files. For a live source such as `rtmp`, you should use `input.rtmp`.

**Reply by ngivoni** [reply]
How would I achieve the following: 

Creating a playlist with files and RTMP playing in a sequential order?  Something like the following:
1. first_file.mp4
2. second_file.mp4
3. RTMP stream
4. third_file.mp4

Also are you available to consult?  If so what is your fee?
